web/styles/popovers.css

```diff
@@ -117,6 +117,17 @@
     --box-shadow-popover-menu: var(--box-shadow-gear-menu);
 }
 
+.modal__title {
```

Comment:
This should be in `modals.css`, probably saving you the need for the heading. Otherwise lgtm.

Reply:
Ah, quite right. Pushed with the correct location.